der/src/asn1/any.rs

```diff
@@ -201,6 +201,14 @@ mod allocating {
         {
             AnyRef::from(self).sequence(f)
         }
+
+        /// [`Any`] representation of the ASN.1 `NULL` type.
+        pub fn null() -> Self {
```

Comment:
This should probably be a `const` or `const fn` if that's possible.

Reply:
`BytesOwned` relies on `Box` which is not `const`

Reply:
Ack